### ISSUE TITLE
[docs] Create documentation for Class constants coding style

### ISSUE THREAD

**Issue by christianabila** [moodle/devdocs#425]
### What is the Moodle feature that needs documenting?

An explicit set of rules for how class constants need to be declared would complete the documentation. So far, it is implied in https://moodledev.io/general/development/policies/codingstyle#constants-1, but with how the document page is currently structured, people might think, that this (https://moodledev.io/general/development/policies/codingstyle#constants) is the way to go.

In conclusion, my request is a subchapter "Class constants" under https://moodledev.io/general/development/policies/codingstyle#classes-1 where the coding style for class constants are explicitly declared.

### Is this documentation specific to a Moodle version?

Unsure (Default)

### Are you able to provide a patch for this?

No (Default)
